src/components/MAPI/apps/InstallIngressButton.tsx

```diff
@@ -65,7 +67,11 @@ const InstallIngressButton: React.FC<
   const appListClient = useRef(clientFactory());
   const appListGetOptions = { namespace: clusterID };
 
-  const appsPermissions = usePermissionsForApps(provider, clusterID);
+  const appsPermissions = usePermissionsForApps(
+    provider,
+    appsNamespace ?? '',
+    isClusterApp
+  );
```

Comment:
Missed updating the permissions hook in `<InstallIngressButton />` earlier - I referenced the app installation PR for the extra props we need to pass in, hopefully it wouldn't create major conflicts